openwisp_radius/base/models.py

```diff
@@ -983,6 +984,10 @@ def save(self, *args, **kwargs):
             self.key = self.generate_key()
         return super().save(*args, **kwargs)
 
+    def delete(self, *args, **kwargs):
+        cache.delete(f'rt-{self.user.username}')
+        super().delete(*args, **kwargs)
```

Comment:
delete is often not called (eg bulk delete or other cases), you could rename this to `delete_cache` and call this method from a post delete signal

Reply:
I covered the bulk delete case from admin, but I'll do this for robustness, thanks.